src/csv.rs

```diff
@@ -78,10 +77,10 @@ pub fn csv(ledger_file: &String, csv_file: &String) -> Result<(), std::io::Error
 
         // loop through transactions and find matching memos
         for transaction in &deserialized_file.transactions {
-            if transaction.name == record.name {
+            if transaction.name.trim() == record.name.trim() {
```

Comment:
Remove some error by trimming names so we ensure that transactions match up.

Reply:
nice solution!

Reply:
Thanks. I came across this during some testing.